pkg/controller/llmisvc/router.go

```diff
@@ -46,25 +46,29 @@ func (r *LLMInferenceServiceReconciler) reconcileRouter(ctx context.Context, llm
 
 	logger.Info("Reconciling Router")
 
+	defer func() {
+		llmSvc.DetermineRouterReadiness()
+	}()
```

Comment:
why not?
```suggestion
	defer llmSvc.DetermineRouterReadiness()
```

Reply:
I think we have some extra params in the test logic so that it's wrapped in the func, if not - we can simplify